takeshape/penny
# Bump deps

### Test Plan (Steps to test):

Check for regressions:

1. Sign in, check all account tabs
1. Check a product page
1. Check a collection page
1. Submit a newsletter signup
